src/runtime/pkg/containerd-shim-v2/shim_management.go

```diff
@@ -148,6 +198,8 @@ func (s *service) startManagementServer(ctx context.Context, ociSpec *specs.Spec
 	m := http.NewServeMux()
 	m.Handle("/metrics", http.HandlerFunc(s.serveMetrics))
 	m.Handle("/agent-url", http.HandlerFunc(s.agentURL))
+	m.Handle("/direct-volume/stats", http.HandlerFunc(s.serveVolumeStats))
+	m.Handle(DirectVolumeResizeUrl, http.HandlerFunc(s.serveVolumeResize))
```

Comment:
Why hardcoded for one and variable for other?

Reply:
updated